app/views/profile/settings.html.erb

```diff
@@ -0,0 +1,77 @@
+<div class="uk-width-medium-1-2 uk-container-center">
+    <%= form_tag({action: :save_settings}, {method: :post}) do %>
+        <table class="uk-table uk-table-striped uk-table-hover">
+            <% for attribute in attributes do %>
+                <tr>
+                    <td><label><%= attribute[:text] %><%= " *" if attribute[:required] %></label></td>
+                    <td>
+                        <%=
+                            if attribute[:type] == :text
```

Comment:
Gondolom ti is erzitek, hogy ez mar most egy code-smell. Az ilyen "okos" kodok szoktak nagyon sok fejfajast okozni. Ez megint egy write-only kodreszlet. Lehet hogy picit tobb karakter lett volna kiirni az attributumokat egyesevel,de akkor nem kene gondolkozni majd, ha egyszer az egyik fieldet valamiert maskepp kell megjeleniteni.